web/src/utils/usePaginatorApi.ts

```diff
@@ -90,12 +90,12 @@ export const usePaginatorApi = <
       setNextToken(data.next);
       // Otherwise incrementing internally between pages
     } else {
-      const tokenIfPresent = tokenStack[pageIndex + 1];
-      if (tokenIfPresent) {
-        setNextToken(tokenStack[pageIndex + 1]);
-      } else {
-        setNextToken(data.next);
-      }
+      // const tokenIfPresent = tokenStack[pageIndex + 1];
+      // if (tokenIfPresent) {
+      //   setNextToken(tokenStack[pageIndex + 1]);
+      // } else {
+      setNextToken(data.next);
+      // }
```

Comment:
This was the offending code. It was pulling old pages from the tokenStack. 

Commenting this out and using the page from the api fixes the issue.